### PR TITLE
introduce version entity

### PR DESCRIPTION
**Pull request by jaapio**

### CODE REVIEW

File: tests/unit/phpDocumentor/Project/Version.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    /**
+     * Return number of this version.
+     *
+     * @return VersionNumber
+     */
+    public function getVersionNumber()
+    {
+        return $this->versionNumber;
```
Comment (mvriel) [inline tests/unit/phpDocumentor/Project/Version.php]: I am missing a test for this method

File: src/phpDocumentor/Project/Version/Factory.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+use phpDocumentor\Project\Version;
+
+/**
+ * Factory to create Version
+ */
+final class Factory
+{
+    /**
+     * @param Definition $definition
```
Comment (mvriel) [inline src/phpDocumentor/Project/Version/Factory.php]: I am missing a description for this DocBlock

**Comment by mvriel** [comment]
Just noticed something: the file tests/unit/phpDocumentor/Project/Version.php seems to contain the class for src/phpDocumentor/Project/Version.php; or am I missing something?

**Comment by jaapio** [comment]
Fixed all your comments, ready for merge

**Comment by mvriel** [comment]
:+1:

**Pull request merged**